api/src/main/java/com/datatorrent/api/DefaultInputPort.java

```diff
@@ -109,4 +109,8 @@ public void teardown()
    */
   public abstract void process(T tuple);
 
+  protected int incrementCount()
```

Comment:
What do you guys think about setting this method to Evolving, at least for one release.

Reply:
OK to mark it as Evolving

Reply:
done